DOCS/man/options.rst

```diff
@@ -1996,6 +1996,13 @@ Audio
 
     Since mpv 0.18.1, this always controls the internal mixer (aka "softvol").
 
+``--ao-volume=<value>``
+    Set the startup system volume. This option is available only if mpv audio
```

Comment:
`available` ?
For options, it should be `It works only if ...` here.

Reply:
done